components/popover/index.js

```diff
@@ -85,6 +91,22 @@ export class Popover extends Component {
 		window[ handler ]( 'scroll', this.throttledSetOffset );
 	}
 
+	focus() {
+		const { content, popover } = this.nodes;
+		if ( ! content ) {
+			return;
+		}
+
+		// Find first tabbable node within content and shift focus, falling
+		// back to the popover panel itself.
+		const firstTabbable = focus.tabbable.find( content )[ 0 ];
```

Comment:
We could try to reuse this utility in `WritingFlow`. I don't know how well this will behave compared to our current selector.

Reply:
>We could try to reuse this utility in `WritingFlow`.

I agree, it would be good to have a consistent, well-tested approach for this.

>I don't know how well this will behave compared to our current selector.

Better (or _more accurate_ anyways), I hope 😬

Reply:
I'll try it ;)